guis/panel/leak/panel_leak_utilities.py

```diff
@@ -92,3 +93,94 @@ def get_time(time_str):
         # df['PSI/degC'] = df["PRESSURE(PSI)"]/df["ROOM TEMPERATURE(C)"]
 
         return df
+
+
+################################################################################
+# READ INPUT
+# Read raw data from the database into a dataframe, structurally equivalent to that of ReadLeakRateFile, but with timestamps retained
+# TIME(DAYS)    FILLPSIA    RefPSIA PRESSURE(PSI)   BOX TEMPERATURE(C)  ROOM TEMPERATURE(C) Heater%
+################################################################################
+
+def readLeakDb(panel, tag):
```

Comment:
I recognize that function naming case conventions are all over the place in this repo, but I have a personal and irrational distaste for `camelCase` used in any situation. For functions, it's either `snake_case` or `UpperCamelCase` for me.
Let's go with a slightly more descriptive name here: `ReadLeakDataFromDB`. And likewise, the other one should be changed to `ReadLeakDataFromFile`. I'd also accept `ReadRawLeakDataFromDB` or even `ReadRawPanelLeakDataFromDB`.